docs/rbac.md

```diff
@@ -0,0 +1,47 @@
+# Run Ark more securely with restrictive RBAC settings
+
+By default Ark runs with an RBAC policy of ClusterRole `cluster-admin`. This is to make sure that Ark can back up or restore anything in your cluster. But `cluster-admin` access is wide open -- it gives Ark components access to everything in your cluster. Depending on your environment and your security needs, you should consider whether to configure more restrictive access. 
+
+**Note:** Roles and RoleBindings are associated with a single namespaces, not with an entire cluster. PersistentVolume backups are associated only with an entire cluster. This means that any backups or restores that use a restrictive Role and RoleBinding pair can manage only the resources that belong to the namespace. You do not need a wide open RBAC policy to manage PersistentVolumes, however. You can configure a ClusterRole and ClusterRoleBinding that allow backups and restores only of PersistentVolumes, not of all objects in the cluster.
```

Comment:
Are you thinking of writing an example ClusterRole and ClusterRoleBinding for PVs?

Another RBAC issue if Ark isn't running as cluster-admin: it can't grant privileges it doesn't currently have. It can't create and bind roles or cluster roles beyond its abilities (https://kubernetes.io/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping).

Reply:
Wasn't planning to write another example for this release, but could do for an interim docs release before 1.0. Added a bit of text to clarify the issue in the meantime.